js/client/modules/@arangodb/testsuites/dump_authentication.js

```diff
@@ -88,7 +96,7 @@ function dumpAuthentication (options) {
 
   let results = { failed: 1 };
   results.setup = tu.runInArangosh(options, instanceInfo,
-    tu.makePathUnix('js/server/tests/dump/dump-authentication-setup.js'),
+    tu.makePathUnix(testPaths.dump_authentication[0]),
```

Comment:
loop or fixed path?!

Reply:
OK then lets have the full path. That will not change the behavior if somethings gets added to the above array or the order is changed. setup, test, tear-down suggests that the order might be important but this is not true foll all tests.